web/react/utils/utils.jsx

```diff
@@ -95,15 +96,20 @@ module.exports.getCookie = function(name) {
   if (parts.length == 2) return parts.pop().split(";").shift();
 }
 
-
 module.exports.notifyMe = function(title, body, channel) {
-  if ("Notification" in window && Notification.permission !== 'denied') {
+    if ("Notification" in window && Notification.permission !== 'denied') {
```

Comment:
Yeah for this and your other comments on the style guide stuff I left a note in the dev room in Cayley that I think may have gotten pushed up, but I was still planning on doing at least one more final pass through for style guide stuff (and removing stuff like the debug statements, etc).